src/Compilers/CSharp/Portable/Generated/CSharp.Generated.g4

```diff
@@ -322,7 +321,11 @@ class_declaration
   ;
 
 extension_declaration
-  : attribute_list* modifier* syntax_token identifier_token type_parameter_list? base_list? type_parameter_constraint_clause* '{'? member_declaration* '}'? ';'?
+  : attribute_list* modifier* ('implicit' | 'explicit') syntax_token identifier_token type_parameter_list? for_type? base_list? type_parameter_constraint_clause* '{'? member_declaration* '}'? ';'?
```

Comment:
ick.  it would be nice if we could update our grammar generator to understand contextial identifiers (like 'extension').  actually.  does it not understand that?  it seems to work for other constructs (like 'from_clause').  so i would expect to see `'extension'` here not `syntax_token`.